src/interop.rs

```diff
@@ -58,10 +59,32 @@ impl From<&tex::Person> for Person {
 impl From<tex::Date> for Date {
     fn from(date: tex::Date) -> Self {
         match date.value {
-            DateValue::At(x) | DateValue::After(x) | DateValue::Before(x) => {
-                Date { year: x.year, month: x.month, day: x.day }
+            DateValue::At(x)
+            | DateValue::After(x)
+            | DateValue::Before(x)
+            | DateValue::Between(_, x) => {
+                let (hour, minute, second, nanosecond) = x
+                    .time
+                    .map(|time| {
+                        (
+                            Some(time.hour() as _),
+                            Some(time.minute() as _),
+                            Some(time.second() as _),
+                            Some(time.nanosecond()),
+                        )
+                    })
+                    .unwrap_or_default();
+                Self {
+                year: x.year,
+                month: x.month,
+                day: x.day,
+                    hour,
+                    minute,
+                    second,
+                    nanosecond,
+                offset: None,
```

Comment:
> 1. At least on Github, this indent looks wonky. Are tabs and spaces mixed here? Best to run rustfmt!

Fixed! VS Code helpfully skipped the whitespace only changes when highlighting the diff.

> 2. This assumes all BibTeX times to be UTC. Does the BibLaTeX manual provide any input on whether to assume the local timezone or UTC? If it assumes the former, does this conversion even make sense to do for our American or East-Asian friends?

Actually, this is leaving the offset unspecified instead of assuming UTC. That is, the resulting `Date` represents a datetime in some timezone, but we do not know which timezone.

According to the documentation of `tex::Date`,  it is not aware of any timezones. So leaving the offset unspecified like this is the best we can do, unfortunately.

The BibLaTeX manual simply states that dates are formatted according to ISO8601-2 Clause 4, level 1 and provides a few examples in table 3, section 2.3.8.

While comparing the implementation of the `biblatex` crate, I noticed that it seems to use a slightly more constrained format, which requires the specification of a full time. Should we adapt the implementation for the YAML format accordingly?